### PR TITLE
Use a background color option, and reverse the logic check;

### PR DESCRIPTION
**Pull request by seangates**
### Changes
- adds the textbox background color to the options
- uses the background color as exclusionary to detect autofill changes

### CODE REVIEW

File: docs/static/bundle.js [hunk 1]
```diff
@@ -601,17 +601,18 @@ function _classCallCheck(instance, Constructor) { if (!(instance instanceof Cons
...
+    labelElementInlineModifier: 'floating-label__label--inline',
+    textboxElementBackgroundRGB: 'rgb(255, 255, 255)'
 };
 
 function hasValue(input) {
     return input.value.length > 0;
 }
 
-function isAutofilled(input) {
+function isAutofilled(input, options) {
```
Comment (ianmcburnie) [inline docs/static/bundle.js]: Maybe just pass in the colour itself as the 2nd param, if all options aren't needed? Otherwise, LGTM!

**Review by DylanPiercey — APPROVED** [review]
Strange hack, but looks okay to me 👍

File: test/static/bundle-module.js [hunk 2]
```diff
@@ -8,17 +8,18 @@ function _classCallCheck(instance, Constructor) { if (!(instance instanceof Cons
...
 function hasValue(input) {
     return input.value.length > 0;
 }
 
-function isAutofilled(input) {
+function isAutofilled(input, options) {
     // check for computed background color because of Chrome autofill bug
     // https://stackoverflow.com/questions/35049555/chrome-autofill-autocomplete-no-value-for-password/35783761#35783761
-    return getComputedStyle(input).backgroundColor === 'rgb(250, 255, 189)';
+    return getComputedStyle(input).backgroundColor !== options.textboxElementBackgroundRGB;
```
Comment (pcanella) [inline test/static/bundle-module.js]: should we check for `options && options.textboxElementBackgroundRGB` here?

Comment (ianmcburnie) [reply test/static/bundle-module.js]: I'd prefer we just pass `textboxElementBackgroundRGB` as the param.

Comment (pcanella) [reply test/static/bundle-module.js]: ^^^ I agree

Comment (seangates) [reply test/static/bundle-module.js]: That's good. I'll change it.

**Comment by seangates** [comment]
@DylanPiercey @pcanella @ianmcburnie Updated addressing comments.

**Review by ianmcburnie — APPROVED** [review]

**Review by ianmcburnie — CHANGES_REQUESTED** [review]
<strike>Actually, can you update the README too?</strike>

**Review by ianmcburnie — APPROVED** [review]
I'll update the README (I notice options section is totally missing anyhow).

**Pull request merged**